doc/devel/contributing.rst

```diff
@@ -30,52 +30,56 @@ You can check out the latest sources with the command::
     git clone git@github.com:matplotlib/matplotlib.git
 
 
-After obtaining a local copy of the matpotlib source code (:ref:`set-up-fork`),
-navigate to the matplotlib directory and run the following in the shell:
-
-::
+After obtaining a local copy of the matplotlib source code (:ref:`set-up-fork`),
+navigate to the matplotlib directory and run the following in the shell::
     
     python setup.py develop
 
 or::
   
    pip install -v -e .
 
-This installs matplotlib for development (i.e., builds everything and places the
-symbolic links back to the source code). You can then run the tests your work
-environment is set up properly::
 
-  python tests.py
+This installs matplotlib for development (i.e., builds everything and places the
+symbolic links back to the source code).
 
+.. warning::
 
-You may want to consider setting up a `virtual environment
-<http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_ or a `conda
-environment <http://conda.pydata.org/docs/using/envs.html>`_
+   If you already have a version of matplotlib installed, you will need to
+   uninstall it.
 
 
 .. note::
   
-    If you decide to do this, you will have to rerun::
+    If you decide to do install with ``python setup.py develop`` or ``pip
+    install -v -e``, you will have to rerun::
    
       python setup.py build
 
     every time the source code of a compiled extension is changed (for
     instance when switching branches or pulling changes from upstream).
 
-.. _nose: http://nose.readthedocs.org/en/latest/
-.. _pep8: http://pep8.readthedocs.org/en/latest/
+
+
+You can then run the tests to check your work environment is set up properly::
+
+  python tests.py
+
+
+.. _nose: https://nose.readthedocs.org/en/latest/
+.. _pep8: https://pep8.readthedocs.org/en/latest/
```

Comment:
Since you're updating these URLs, would this be a good time to move to readthedocs.io? (See [here](http://blog.readthedocs.com/securing-subdomains/)